src/library/psi_library.jl

```diff
@@ -1291,7 +1291,6 @@ function _duplicate_system(main_sys::PSY.System, twin_sys::PSY.System, HVDC_line
 
         IS.assign_new_uuid!(twin_sys.data, b)
         PSY.remove_component!(twin_sys, b)
-        b.time_series_container = IS.TimeSeriesContainer()
```

Comment:
@jd-lara I don't think this should have been there. Let me know if there was a reason for it.